hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/interpolation/MessageBodyInterpolatorTest.java

```diff
@@ -102,8 +115,16 @@ public void willInterpolateMultipleJsonPathsFromTemplateInReverseOrder() throws
         EndpointAddress endpoint = EndpointAddress.of("http://some.endpoint.com/{some.object}?test={some.test}");
         URI expectedEndpoint = URI.create("http://some.endpoint.com/100?test=hello");
         String jsonMessage = "{\"some\": {\"test\": \"hello\", \"object\": 100}}";
-        Message msg = new Message("id", "some.topic", jsonMessage.getBytes(), Topic.ContentType.JSON, 1232443L, 12324434L,
-                new PartitionOffset(KAFKA_TOPIC, 0, 0));
+//        Message msg = withTestMessage()
```

Comment:
Here are some leftovers to clean up. I like that you wrote the builder, maybe we can also use it here.